### PR TITLE
Refactor organisms to be async / await + fix stability #trivial

### PR DESCRIPTION
**Pull request by siggerzz**

### CODE REVIEW

File: packages/components/organisms/f-content-cards/test/component/f-content-cards-home-promotion-card1.component.spec.js [hunk 1]
```diff
@@ -1,48 +1,25 @@
...
-        await expect(await card.isHeaderDisplayed()).toBe(true);
-        await expect(await card.isImageDisplayed()).toBe(true);
-        await expect(await card.isSubtitleDisplayed()).toBe(true);
-    });
-
-    it('should display the elements of the inner card', async () => {
-        // Why does this pass when the second card was not loaded?
-        // Because loading card is enough to make these tests pass?
-        // Or something more nefarious??!
-        // Assert
```
Comment (siggerzz) [inline packages/components/organisms/f-content-cards/test/component/f-content-cards-home-promotion-card1.component.spec.js:31]: Removed this test from this file as it's covered in another spec.

File: packages/components/organisms/f-content-cards/test/component/f-content-cards-home-promotion-card1.component.spec.js [hunk 2]
```diff
@@ -1,48 +1,25 @@
...
     });
 
     it('should check that the card is clickable', async () => {
         // Assert
-        await expect(await card.isComponentClickable()).toBe(true);
-    });
-
-    it('should check that the inner-card is clickable', async () => {
-        // Assert
-        await expect(await card2.isComponentClickable()).toBe(true);
```
Comment (siggerzz) [inline packages/components/organisms/f-content-cards/test/component/f-content-cards-home-promotion-card1.component.spec.js:46]: ditto

File: packages/components/organisms/f-cookie-banner/test-utils/component-objects/f-cookie-banner-legacy.component.js [hunk 3]
```diff
@@ -9,15 +9,15 @@ module.exports = class CookieBannerLegacy extends Page {
 
     get cookiePolicyLink () { return $('[data-test-id="cookie-policy-link"]'); }
 
-    get cookiePolicyLinkHref () { return this.cookiePolicyLink.getAttribute('href'); }
-
     get closeButton () { return $('[data-test-id="cookieBanner-close-button"]'); }
 
-    clickCookiePolicyLink () { this.cookiePolicyLink.click(); }
+    async cookiePolicyLinkHref () { return this.cookiePolicyLink.getAttribute('href'); }
```
Comment (siggerzz) [inline packages/components/organisms/f-cookie-banner/test-utils/component-objects/f-cookie-banner-legacy.component.js:14]: Changed this to a function due to `getAttribute` being a WDIO async function, so can no longer be a getter.

File: packages/components/organisms/f-cookie-banner/test/accessibility/axe-accessibility.spec.js [hunk 4]
```diff
@@ -1,61 +1,50 @@
...
-        const expectedTabOrder = [
-            cookieBanner.cookiePolicyLink,
-            cookieBanner.cookieAcceptAllButton,
-            cookieBanner.cookieAcceptNecessaryButton,
-            cookieBanner.cookieBannerTitle];
-        const result = cookieBanner.testTabOrder(expectedTabOrder);
-        expect(result.actual).toEqual(result.expected);
+
+
+    const tenants = [
```
Comment (siggerzz) [inline packages/components/organisms/f-cookie-banner/test/accessibility/axe-accessibility.spec.js:28]: Removed all occurrences of `mocha-each` as it was causing flakiness with async. This new approach is recommended by mocha - https://mochajs.org/#dynamically-generating-tests

File: packages/components/organisms/f-footer/test/visual/f-footer.visual.spec.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
+import Footer from '../../test-utils/component-objects/f-footer.component';
```
Comment (siggerzz) [inline packages/components/organisms/f-footer/test/visual/f-footer.visual.spec.js]: Old desktop / mobile tests were deleted and both moved into the same file to match other components.

**Review by JoshuaNg2332 — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**